### ISSUE TITLE
Memory leak in rrdtool.info

### ISSUE THREAD

**Issue by hvdovic** [commx/python-rrdtool#39]
There is an issue with the `rrdtool.info` function which causes a memory leak. This was already reported in #25 and closed, but it is not fixed. I managed to reproduce the issue locally by running the test script provided in #25 and got similar values in valgrind report.

Test script:
```
import rrdtool
for i in range(1000):
    rrdtool.info("file.rrd")
```

Valgrind report:
```
==1957== HEAP SUMMARY:
==1957==     in use at exit: 771,052 bytes in 683 blocks
==1957==   total heap usage: 381,965 allocs, 381,282 frees, 35,060,855 bytes allocated
==1957== 
==1957== LEAK SUMMARY:
==1957==    definitely lost: 0 bytes in 0 blocks
==1957==    indirectly lost: 0 bytes in 0 blocks
==1957==      possibly lost: 7,296 bytes in 29 blocks
==1957==    still reachable: 763,756 bytes in 654 blocks
==1957==                       of which reachable via heuristic:
==1957==                         newarray           : 1,536 bytes in 16 blocks
==1957==         suppressed: 0 bytes in 0 blocks
==1957== Rerun with --leak-check=full to see details of leaked memory
==1957== 
```

If the test script is run in an infinite loop, the memory usage keeps rising until no more available memory is left.

`rrdtool.lib_version()`: `1.7.1`
`python-rrdtool`: `0.1.15`
`python`: `3.7.3`

**Comment by stephan-hof** [comment]
This leak is fixed with the following commit:
https://github.com/commx/python-rrdtool/commit/273f2ccbdb20e4d83ff3987385d6823b6ba9d933

Why ?
By having the return from `PyFloat_FromDouble` assigned to `val` its reference count gets decreased later here:
```
Py_DECREF(val);
```
Without this fix the return value from `PyFloat_FromDouble` lives forever, because nobody decreased its ref-count.

Unfortunately this commit is only in *master*. So could somebody please publish a new release on pypi containing this fix?

By using master a script like this ran for hours without leaking.
```
import rrdtool
while True:
    rrdtool.info("file.rrd")
```

**Comment by commx** [comment]
Hi, thanks for the feedback. A new version ([0.1.16](https://pypi.org/manage/project/rrdtool/release/0.1.16/)) has been released and published on PyPI. Let me know if this fixes your issue.

**Comment by hvdovic** [comment]
I can confirm that this issue is fixed in version 0.1.16. Thanks!
